### PR TITLE
Add `open-issue-to-latest-comment` feature

### LINKED ISSUE #1969: Open issue to latest comment
Sometimes I'm only interested in the latest comment. What if the comment number is linked to the _latest_ comment...

image

**Test:**
https://github.com/sindresorhus/refined-github/issues for issues with multiple comments.

### PR DESCRIPTION
**Pull request by dotconnor**
Closes #1969

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -3,3 +3,4 @@ source/features/tags-dropdown.tsx @HardikModha
 source/features/tag-changelog-link.tsx @HardikModha
 source/features/link-to-file-in-file-history.tsx @HardikModha
 source/features/default-to-rich-diff.tsx @idasbiste
+source/features/open-issue-to--comment.tsx @dotconnor
```
Comment (bfred-it) [inline .github/CODEOWNERS]: 👆

File: source/features/open-issue-to-latest-comment.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoPath, getOwnerAndRepo} from '../libs/utils';
+
+function init(): void | false {
+	if (getRepoPath() === 'issues') {
+		const {ownerName, repoName} = getOwnerAndRepo();
+		select.all<HTMLAnchorElement>(`.js-issue-row a.muted-link[href^="/${ownerName}/${repoName}/issues/"]`).forEach(link => {
+			link.href = `${link.href}#last`;
```
Comment (bfred-it) [inline source/features/open-issue-to-latest-comment.tsx]: You can set `link.hash` and then link to the footer. Refer to the link in the old `jump-to-bottom` feature

File: source/features/open-issue-to-latest-comment.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+			}
+		}, 100);
+	}
+}
+
+features.add({
+	id: 'open-issue-to-latest-comment',
+	include: [
+		features.isIssue,
+		features.isIssueList
```
Comment (bfred-it) [inline source/features/open-issue-to-latest-comment.tsx]: Only use `isDiscussionList`

File: source/features/open-issue-to-latest-comment.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoPath, getOwnerAndRepo} from '../libs/utils';
+
+function init(): void | false {
+	if (getRepoPath() === 'issues') {
+		const {ownerName, repoName} = getOwnerAndRepo();
+		select.all<HTMLAnchorElement>(`.js-issue-row a.muted-link[href^="/${ownerName}/${repoName}/issues/"]`).forEach(link => {
```
Comment (bfred-it) [inline source/features/open-issue-to-latest-comment.tsx]: Are you sure you need this selector?

Also use for-of

File: source/features/open-issue-to-latest-comment.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void | false {
+	for (const link of select.all<HTMLAnchorElement>('a[aria-label*="comment"]')) {
+		link.href = `${link.href}#partial-timeline`;
```
Comment (bfred-it) [inline source/features/open-issue-to-latest-comment.tsx]:
```suggestion
		link.hash = '#partial-timeline';
```

File: source/libs/page-detect.ts [hunk 6]
```diff
@@ -29,6 +29,8 @@ export const isGlobalSearchResults = (): boolean => location.pathname === '/sear
 
 export const isIssue = (): boolean => /^issues\/\d+/.test(getRepoPath()!);
 
+export const isIssueList = (): boolean => getRepoPath() === 'issues';
+
```
Comment (bfred-it) [inline source/libs/page-detect.ts]: This can be dropped

File: source/features/open-issue-to-latest-comment.tsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+function init(): void | false {
```
Comment (bfred-it) [inline source/features/open-issue-to-latest-comment.tsx]:
```suggestion
function init(): void {
```

**Comment by bfred-it** [comment]
I think this is almost good to go:

- [ ] Merge with master, it needs a description field because of #2030 
- [ ] Mention the feature in the readme under the shortcuts

**Comment by bfred-it** [comment]
Once these are in it can be merged

File: readme.md [hunk 8]
```diff
@@ -206,6 +206,7 @@ GitHub Enterprise is also supported. More info in the options.
 - [Search or select tags from a dropdown in the `Releases` page.](https://user-images.githubusercontent.com/22439276/56373231-27ee9980-621e-11e9-9b21-601919d3dddf.png)
 - [Link to file itself in the history pages](https://user-images.githubusercontent.com/22439276/57195061-b88ddf00-6f6b-11e9-8ad9-13225d09266d.png)
 - [See an automatic changelog for each tag or release.](https://user-images.githubusercontent.com/1402241/57081611-ad4a7180-6d27-11e9-9cb6-c54ec1ac18bb.png)
+- [Quickly open to the latest comment](https://user-images.githubusercontent.com/14323370/57660122-02517780-75b3-11e9-9370-4c0426b248ca.gif)
```
[the remark this answers is not in the document]
Comment (bfred-it) [reply readme.md]: This is still a gif

[the remark this answers is not in the document]
Comment (dotconnor) [reply readme.md]: My bad, I thought the suggestion was just the description change.

**Comment by bfred-it** [comment]
Thank you 🙏

**Pull request merged**